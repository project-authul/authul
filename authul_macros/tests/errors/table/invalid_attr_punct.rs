#[authul_macros::authul_table(name +)]
struct Foo;

#[authul_macros::authul_table(name !=)]
struct Bar;

#[authul_macros::authul_table(name ==)]
struct Baz;

fn main() {
}
