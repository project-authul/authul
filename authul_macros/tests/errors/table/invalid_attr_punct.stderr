error: expected '='
 --> tests/errors/table/invalid_attr_punct.rs:1:36
  |
1 | #[authul_macros::authul_table(name +)]
  |                                    ^

error: expected '='
 --> tests/errors/table/invalid_attr_punct.rs:4:36
  |
4 | #[authul_macros::authul_table(name !=)]
  |                                    ^

error: expected '='
 --> tests/errors/table/invalid_attr_punct.rs:7:36
  |
7 | #[authul_macros::authul_table(name ==)]
  |                                    ^
